### PR TITLE
Specify icon with file extension

### LINKED ISSUE #320: Icon is broken on Open Store Website
The icon is not shown on the [Open Store Website](https://open-store.io/app/pure-maps.jonnius).  This is probably caused by the icon not having a file extension in the click package.

### PR DESCRIPTION
**Pull request by jonnius**
Fixes #320 and increments version patch for the patch release on the open store that also fixes pasting API keys.

### CODE REVIEW

File: packaging/ubports/manifest.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
             "desktop":  "pure-maps.desktop"
         }
     },
-    "version": "1.24.0",
+    "version": "1.24.1",
```
Comment (rinigus) [inline packaging/ubports/manifest.json]: let's put 1.24.0.1 here. I am sometimes using third number to update translations or fast bug fixes

Comment (jonnius) [reply packaging/ubports/manifest.json]: I have already released it like that. :see_no_evil:

Comment (jonnius) [reply packaging/ubports/manifest.json]: Next time I'll append a forth number if I do a Ubuntu Touch only patch release...

Comment (rinigus) [reply packaging/ubports/manifest.json]: OK, no problem. let's just keep it in mind. although, I hope we will not need any special UT releases :)

**Pull request merged**